nailgun/entity_fields.py

```diff
@@ -265,4 +266,4 @@ class URLField(StringField):
 
     def gen_value(self):
         """Return a value suitable for a :class:`URLField`."""
-        return gen_url()
+        return gen_url(subdomain=gen_alpha())
```

Comment:
@rplevka You're right. It'd be good to generate more "interesting" data. I didn't both pushing for the change, though, because this PR really just works around issues in the underlying fauxfactory library. An even better solution is to make the `gen_url` function produce a greater variety of URLs and obey [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.txt).